api/migrations/env.py

```diff
@@ -17,24 +15,45 @@
 fileConfig(config.config_file_name)
 logger = logging.getLogger("alembic.env")
 
+
+def get_engine():
```

Comment:
Most of the changes in this file are boilerplate changes when I regenerated the contents using `flask db init`. You can safely ignore them except for stuff I've specifically called out.